homeassistant/components/sensor/pollen.py

```diff
@@ -185,6 +193,11 @@ def state(self):
         """Return the state."""
         return self._state
 
+    @property
+    def unique_id(self):
+        """Return a unique, HASS-friendly identifier for this entity."""
+        return '{0}_{1}'.format(self._entity_id, slugify(self._name))
```

Comment:
Names are configurable and thus not unique.

Reply:
"Name" is a misnomer here: those are defined within `CONDITIONS` (example: `Allergy Index: Forecasted Average`). The user cannot change them.